log10/feedback/feedback.py

```diff
@@ -111,3 +160,40 @@ def _get_feedback_list(offset, limit, task_id):
         return []
 
     return feedback_data
+
+
+def _format_graphql_node(node):
+    return {
+        "id": node["id"],
+        "json_values": node["jsonValues"],
+        "task_id": node["task"]["id"],
+        "task_name": node["task"]["name"],
+        "matched_completion_ids": [c["id"] for c in node["completions"]],
+    }
+
+
+def _get_feedback_list_graphql(page, limit, task_id, filter):
+    feedback_data = []
+    current_page = page if page else 1
+    if limit:
+        limit = int(limit)
+
+    try:
+        while True:
+            fetch_limit = limit if limit else 50
+            res = Feedback().list_v2(page=current_page, limit=fetch_limit, task_id=task_id, filter=filter)
+            new_data = res["data"]["organization"]["feedbackV2"]["nodes"]
+            feedback_data.extend(new_data)
+
+            current_fetched = len(new_data)
+            current_page += 1
+
+            if current_fetched < fetch_limit:
+                break
+    except Exception as e:
+        logger.error(f"Error fetching feedback {e}")
+        if hasattr(e, "response") and hasattr(e.response, "json") and "error" in e.response.json():
+            logger.error(e.response.json()["error"])
+        return []
+
+    return list(map(_format_graphql_node, feedback_data))
```

Comment:
```suggestion
    return [_format_graphql_node(item) for item in feedback_data]
```
more readable in python sense. Also I realized that we have used `list` as a function in `Feedback` class. We should do a follow up to rename that function, so that we don't name function the same as python build-in function.